src/components/views/rooms/EventTile.js

```diff
@@ -50,17 +50,6 @@ var eventTileTypes = {
 
 var MAX_READ_AVATARS = 5;
 
-// Our component structure for EventTiles on the timeline is:
```

Comment:
The point of the comment wasn't to mock up the presentation or physical layout, but show the logical structure of the nested components - i.e. what the difference is between a FooBody and an FooEvent and an EventTile.  Thanks for putting it back.